docs/language/learn-ql/introduction-to-ql.rst

```diff
@@ -114,15 +110,15 @@ To simplify the query, we can introduce a class ``SmallInt`` representing the in
 
 ➤ `See this in the query console on LGTM.com <https://lgtm.com/query/2101340747/>`__
 
-Now that you've seen some general examples, let's use the CodeQL libraries to analyze projects.
-In particular, LGTM generates a database representing the code and then CodeQL is used to query this database. For more information, see `Database generation <https://lgtm.com/help/lgtm/generate-database>`__ on LGTM.com.
+Example CodeQL queries
+----------------------
 
-.. XX: Perhaps a link to the "CodeQL libraries for X"?
+The previous examples used the primitive types built in to QL. Although we chose a project to query, they did not use the project-specific database. The following example queries *do* use these databases and give you an idea of how CodeQL is used to analyze projects.
```

Comment:
It's a good call to rewrite this, but I think it might want a little adjusting. Although, having said that, you probably have a better handle on how much detail we actually want to go into here.

The previous queries gave results for the QL language, rather than searching for results in the chosen project database, and they didn't use the library that we imported. I'm not sure how much of this to put in here. What do you think?

*EDITED: Just realized that this is existing text, in which case, you should ignore me*

Reply:
You're right, this wasn't a very helpful paragraph - I've tweaked it a little. 
(A lot of the articles are still very LGTM-centric, so there's probably more work to do in the long run!)